package.json

```diff
@@ -247,13 +247,13 @@
     "@types/semver": "^7.3.10",
     "@types/ssri": "^7.1.1",
     "@types/uuid": "^9.0.5",
-    "@vitejs/plugin-react": "^4.3.4",
-    "@vitejs/plugin-react-swc": "^3.8.0",
```

Comment:
Not compatible with compiler just yet, see https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react-swc/CHANGELOG.md vs https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/CHANGELOG.md

What we lose here is dev server cold start and HMR speeds, though in effect haven't really noticed a diff with cold start.

HMR speed isn't an issue either, with one large caveat: cold start (basically `yarn dev`) can take a long time i.e more than 1:30 to be in an HMR-ready state.
Once there, HMR does work as fast as before however.